libs/androidlib/src/mill/androidlib/AndroidSdkModule.scala

```diff
@@ -47,6 +47,16 @@ trait AndroidSdkModule extends Module {
    */
   def buildToolsVersion: T[String]
 
+  /**
+   * Specifies the version of the Android NDK (Native Development Kit) to be used.
+   */
+  def ndkVersion: T[String] = Task { "27.0.12077973" }
```

Comment:
add a todo to these versions to add them to the build info later on and not have them hardcoded here, I'll do that in a follow up PR